kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -78,6 +87,8 @@ public KafkaProxyInitializer(FilterChainFactory filterChainFactory,
         this.bindingResolver = bindingResolver;
         this.filterChainFactory = filterChainFactory;
         this.apiVersionsService = apiVersionsService;
+        List<Tag> newTags = Metrics.tags(VIRTUAL_CLUSTER_LABEL, "null", NODE_ID_LABEL, "null");
```

Comment:
@k-wall Hi, I tried to pass an empty list here but it seems we can't have meters with same name and different tags

Reply:
The `preventDifferentTagNameRegistration` seems to exist because of some limitation of promethues -> 
```
    /**
     * By default, when we register multiple meters with the same name but different tag names,
     * only the data for one of those meters can be scraped at the prometheus endpoint.
     * This is a limitation of the prometheus client. So we add a filter that explodes when we
     * attempt to register a metric with a different set of tags.
```

Reply:
I can't pass empty values to the the tags :(, another things which I discovered yesterday. The cause is the `required` method in `Metrics` class. 
```
    @NonNull
    private static String required(String value) {
        if ((Objects.isNull(value) || value.trim().isEmpty())) {
            throw new IllegalArgumentException("tag value supplied without a value");
        }
        return value;
    }
```

If its fine I can change the method above to not check for `isEmpty()` i.e. if I pass just "".

Reply:
Does the `MeterProvider` interface being used in https://github.com/kroxylicious/kroxylicious/pull/2243 start to address this?

Reply:
Background: https://github.com/prometheus/client_java/issues/696

Reply:
> If its fine I can change the method above to not check for `isEmpty()` i.e. if I pass just "".

Yes, that's okay.  I have a feeling we won't be using the the method anyway.